data-connector-lib/Makefile

```diff
@@ -2,6 +2,8 @@
 REPOROOT=..
 include $(REPOROOT)/.make.defaults
 
+DPK_CONNECTOR_VERSION=0.2.2
```

Comment:
this should be moved to .make.versions and incremented to 0.2.3.dev0 before re-running set-versions